app/main.py

```diff
@@ -1 +1,59 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.name = name
+        self.skills = ["JavaScript", "HTML", "CSS", ]
```

Comment:
Do not redefine the list for `self.skills` defined in the parent class. Extend it with new values using `+=` or `.extend()` method instead. Apply this change to the `BackendDeveloper` and `AndroidDeveloper` classes as well.